libyul/backends/evm/ControlFlowGraphBuilder.cpp

```diff
@@ -0,0 +1,484 @@
+/*
+	This file is part of solidity.
+
+	solidity is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	solidity is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with solidity.  If not, see <http://www.gnu.org/licenses/>.
+*/
+// SPDX-License-Identifier: GPL-3.0
+/**
+ * Transformation of a Yul AST into a control flow graph.
+ */
+
+#include <libyul/backends/evm/ControlFlowGraphBuilder.h>
+#include <libyul/AST.h>
+#include <libyul/Exceptions.h>
+#include <libyul/Utilities.h>
+#include <libyul/AsmPrinter.h>
+
+#include <libsolutil/cxx20.h>
+#include <libsolutil/Visitor.h>
+#include <libsolutil/Algorithms.h>
+
+#include <range/v3/action/push_back.hpp>
+#include <range/v3/action/erase.hpp>
+#include <range/v3/range/conversion.hpp>
+#include <range/v3/view/concat.hpp>
+#include <range/v3/view/drop_last.hpp>
+#include <range/v3/view/enumerate.hpp>
+#include <range/v3/view/filter.hpp>
+#include <range/v3/view/iota.hpp>
+#include <range/v3/view/map.hpp>
+#include <range/v3/view/reverse.hpp>
+#include <range/v3/view/single.hpp>
+#include <range/v3/view/take_last.hpp>
+#include <range/v3/view/transform.hpp>
```

Comment:
Maybe just include the whole thing? :p

Reply:
The whole thing is huge :-D. I do that at times to write small test snippets and even for those it noticably sucks for compile time :-D.